### DISCUSSION TITLE
Move "Share..." option to device view

### DISCUSSION THREAD

**Discussion by SnoutBug** [andyholmes/valent#729] [Feature Request]
When using the shell extension for valent, the device view is opened when clicking on a device.
Since you can not share from this view directly, you have to navigate back to the device overview in order to select share and then reselect the device again to which you want to share.

I think this is counter intuitive and not very user friendly.

**Comment by andyholmes** [comment]
There's a good chance the device buttons will be removed form the extension, but regardless the "Share..." item was intentionally moved out of the device page.

Once I have more time to think through the main window, the action will probably move out of the menu and be a semi-dynamic button.
